README.md

```diff
@@ -3,7 +3,7 @@
 
 **Qiskit** is an open-source framework for working with noisy quantum computers at the level of pulses, circuits, and algorithms.
 
-Qiskit is made up of elements that work together to enable quantum computing. This element is **Terra** and is the foundation on which the rest of Qiskit is built.
+The core component of Qiskit is `qiskit-terra`.
```

Comment:
If we're going to give the full package name in code format, it might not hurt to say that it supplies the `qiskit` namespace - I think this is a fairly common source of confusion for new users (and experienced ones, sometimes).  Alternatively, we could avoid `` `qiskit-terra` `` and just write "Terra", similar to how we had it before?

Reply:
good point.
```suggestion
The core component of Qiskit is **Terra**.
```